40502.9 <c-14#213> Created logger
40502.9 <c-14#213:17> Logger was updated
40502.9 <c-14#213:17> Started intializing cube lib
40502.9 <c-14#213:17> Finished intializing cube lib with worker
40502.9 <c-14#213:17> shareCubes is called
40502.9 <c-14#213:17> Added 0 failed local cubes
40502.9 <c-14#213:17> Pruned 0 local cubes
40502.9 <c-14#213:17> CubeSolverThread entered cube retrieval loop, currently there are 0 cubes
40502.9 <c-14#213:17> returnFailedAndRequestCubes => [54] {}
40502.9 <c-14#213:17> Sent 0 failed cubes to root and requested new cubes
40502.9 <c-14#213:17> Received 4 cubes from root
40502.9 <c-14#213:17> CubeSolverThread entered cube retrieval loop, currently there are 4 cubes
40502.9 <c-14#213:17> CubeSolverThread retrieved 4 cubes
40502.9 <c-14#213:17> Started solving a cube
40502.9 <c-14#213:17> Cube failed
40502.9 <c-14#213:17> Added failed cube
40502.9 <c-14#213:17> Started solving a cube
40503.6 <c-14#213:17> Cube failed
40503.6 <c-14#213:17> Added failed cube
40503.6 <c-14#213:17> Started solving a cube
40503.7 <c-14#213:17> Cube failed
40503.7 <c-14#213:17> Added failed cube
40503.7 <c-14#213:17> Started solving a cube
40532.9 <c-14#213:17> appl_pause was called
40532.9 <c-14#213:17> SUSPEND (0.00s since last cb)
40703.9 <c-14#213:17> appl_interrupt was called
40703.9 <c-14#213:17> RESUME
40703.9 <c-14#213:17> STOP after suspension
40703.9 <c-14#213:17> Solving interrupted
40703.9 <c-14#213:17> Leaving the main loop
40703.9 <c-14#213:17> appl_interrupt was called
40703.9 <c-14#213:17> appl_withdraw was called
40703.9 <c-14#213:17> Started withdraw thread
40704 <c-14#213:17> Enter destructor of CubeWorkerInterface
40704 <c-14#213:17> Enter destructor of CubeRoot
40704 <c-14#213:17> Finished withdraw thread
40764.9 <c-14#213:17> Enter destructor
40764.9 <c-14#213:17> Time working: 30.070
40764.9 <c-14#213:17> Time suspended: 171.004
40764.9 <c-14#213:17> Joined cleanup thread
40764.9 <c-14#213:17> Exit destructor
40764.9 <c-14#213:17> Destructing logger
